src/qiskit_sphinx_theme/assets/styles/furo/_index.sass

```diff
@@ -0,0 +1,19 @@
+/* This file is vendored from Furo (created by Pradyun Gedam) and used under the MIT license.
+ *
+ * It's only changes are:
+ *
+ *   1. Renaming the file from `furo.sass` to `_index.sass`.
+ *   2. Including furo-extensions.sass directly in it.
+ */
```

Comment:
This comment is worth reading. 

But every other file in `assets/styles/furo` is 100% identical to Furo outside of the copyright comment at the top.